src/storage/cdb/cdb_ffi/src/lib.rs

```diff
@@ -92,8 +132,8 @@ pub extern "C" fn cdb_get(
                 let mut vstr = val.as_mut();
                 vstr.len = n as u32;
             }
-            val.into_raw()  // consume BufStringRefMut and return the underlying raw pointer
-        },
+            val.into_raw() // consume BufStringRefMut and return the underlying raw pointer
+        }
         Ok(None) => ptr::null_mut(), // not found, return a NULL
         Err(err) => {
             eprintln!("got error: {:?}", err);
```

Comment:
At some point we need to consider the choice of logger for debug information, hopefully it'd be possible to stay consistent with the C portion of pelikan.

Reply:
It may be possible to hook the log crate into the existing pelikan log machinery by writing our own backend for it. I'll look into it.